tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -49,7 +49,6 @@ var excludedMetrics = map[string]struct{}{
 	"kubevirt_virt_operator_leading_total":                 struct{}{},
 	"kubevirt_virt_operator_ready_total":                   struct{}{},
 	"kubevirt_virt_operator_up_total":                      struct{}{},
-	"kubevirt_vmi_cpu_affinity":                            struct{}{},
```

Comment:
@machadovilaca You dont need to add the new metric name here?

Reply:
not really, if it was needed `pull-kubevirt-generate` job would have failed
`kubevirt_vmi_cpu_affinity` was probably not needed there either